service/service.go

```diff
@@ -199,20 +199,20 @@ func New(config Config) (*Service, error) {
 			ResourceNamespace:  resourceNamespace,
 		}
 
-		azureClusterController, err = azure.NewCluster(c)
+		azureClusterController, err = azure.NewLegacyCluster(c)
 		if err != nil {
 			return nil, microerror.Mask(err)
 		}
 	}
 
-	var kvmClusterController *kvm.Cluster
+	var kvmClusterController *kvm.LegacyCluster
```

Comment:
I think the variable names should be aligned as well as we are going to name the other controllers somehow. What was your idea there?

Reply:
Didn't outright have an idea. There should be only one cluster controller later, but yeah, it makes sense to align these. One minute, please. :slightly_smiling_face: